bears/generate_package.py

```diff
@@ -51,8 +52,11 @@ def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
     upload_package_folder = os.path.join(root_folder, object_name, object_name)
     os.makedirs(upload_package_folder, exist_ok=True)
     touch(os.path.join(upload_package_folder, '__init__.py'))
-    shutil.copyfile(file_to_copy, os.path.join(upload_package_folder,
-                                               object_name + '.py'))
+    try:
+        shutil.copyfile(file_to_copy, os.path.join(upload_package_folder,
+                                                   object_name + '.py'))
+    except shutil.SameFileError:
+        pass
```

Comment:
this is here because since i changed the new versioning system, some bears fail to upload saying that its the same file or whatever, so I need this until i find out why some bears are geting uploaded multiple times.